flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/TableEnvironment.scala

```diff
@@ -719,33 +715,47 @@ abstract class TableEnvironment(val config: TableConfig) {
       throw new TableException("Field name can not be '*'.")
     }
 
-    (fieldNames.toArray, fieldIndexes.toArray)
+    (fieldNames.toArray, fieldIndexes.toArray) // build fails if not converted to array
```

Comment:
See https://travis-ci.org/apache/flink/jobs/263806347
It only happens when using Scala 2.10.